src/stpipe/log.py

```diff
@@ -192,6 +193,8 @@ def _level_check(value):
         return value
 
     spec = config_parser.load_spec_file(LogConfig)
+    if isinstance(config_file, pathlib.Path):
+        config_file = str(config_file)
     config = ConfigObj(config_file, raise_errors=True, interpolation=False)
```

Comment:
What do you think of the following as an alternative to the `isinstance` conditional?
```suggestion
    config = ConfigObj(str(config_file), raise_errors=True, interpolation=False)
```

Reply:
Thanks for checking. It looks like I missed that `config_file` might also be a file-like object.

Reply:
Ah, and that's why in the test when we send it an `io.StringIO` instance, it barfs.  Thanks.  I'll keep the conditional here then.

Though honestly, I think passing around bytes objects in the logger is bit, um, old-fashioned.  I will leave this mess to future refactoring.  =)

Reply:
I will revert it in the branch.